### DISCUSSION TITLE
Integrating Litellm with Oracle Generative AI

### DISCUSSION THREAD

**Discussion by octadion** [BerriAI/litellm#5878] [Q&A]
Hi everyone,

I’m looking to integrate Litellm with Oracle Generative AI, but from what I’ve seen in the documentation, it seems like this is not yet supported. Are there any workarounds or alternative methods I can try to make this integration work? Or is it currently not possible at all?

Thanks!

**Comment by akscipy** [comment]
I am also looking for the same due to crewai latest change
